src/index.js

```diff
@@ -39,4 +46,60 @@ myPromise.then((result) => {
         <p class="reservations">Reservations</p>  
         </div>`;
   });
+  const commentbutton = document.querySelectorAll('.comments');
+  commentbutton.forEach((element, index) => {
+    element.addEventListener('click', () => {
+      const mypromisec = new Promise((resolve) => {
+        resolve(getcomments(comments, index));
+      });
+      const popupinfo = getmovie(result, popup, index);
```

Comment:
2.Use const instead of let when declaring variables that won't be reassigned.
``` javascript
const myPromise = new Promise((resolve) => {
  resolve(getComments(comments, index));
});
```